elasticsearch/docker/build/elasticsearch/bin/docker-entrypoint.sh

```diff
@@ -92,9 +92,11 @@ if [[ -d "/usr/share/elasticsearch/plugins/opendistro-performance-analyzer" ]];
     CLK_TCK=`/usr/bin/getconf CLK_TCK`
     ES_JAVA_OPTS="-Dclk.tck=$CLK_TCK -Djdk.attach.allowAttachSelf=true $ES_JAVA_OPTS"
     if [[ -d "/usr/share/elasticsearch/performance-analyzer-rca" ]]; then
+        sed -i "s/^user=.*/user=${PUID:-${DEFAULT_UID}}/" /usr/share/elasticsearch/performance-analyzer-rca/pa_config/supervisord.conf
```

Comment:
the performance analyzer's supervisord.conf file is hard-coded with `user=1000`. Use sed to alter its contents in-place and set the user UID to PUID, defaulting to DEFAULT_UID